src/helpers/clipboard/clipboard_helpers.ts

```diff
@@ -54,3 +61,20 @@ export function getPasteZones<T>(target: Zone[], content: T[][]): Zone[] {
     height = content.length;
   return target.map((t) => splitZoneForPaste(t, width, height)).flat();
 }
+
+export function gePastablePluginClipBoardContent(
+  content: OSClipboardContent
+): ImportClipboardContent {
+  const htmlDocument = new DOMParser().parseFromString(
+    content[ClipboardMIMEType.Html] ?? "<div></div>",
+    "text/html"
+  );
+  const oSheetClipboardData = htmlDocument
+    .querySelector("div")
+    ?.getAttribute("data-osheet-clipboard");
+  const spreadsheetContent = oSheetClipboardData && JSON.parse(oSheetClipboardData);
+  return {
+    [ClipboardMIMEType.PlainText]: content[ClipboardMIMEType.PlainText],
+    [ClipboardMIMEType.Html]: spreadsheetContent,
```

Comment:
I would change the HTML tag for somethnin else at this point, it's juste confusing to reference HTML content when it's just a JSON.

Also we might wanna use the actual HTML in the future, if we want to support copying a table from some webpage or things like that

Reply:
yup I agree, I'm trying to come up with a better approach as it gets even funkier with the images

Reply:
note to self: change for text, data (and in the next PR, add a key image)